{{cookiecutter.project_slug}}/clients/web/vue3/src/components/NavBar.vue

```diff
@@ -145,11 +145,19 @@ export default {
     let profileMenuOpen = ref(false)
 
     async function logout() {
-      await userApi.csc.logout()
-      profileMenuOpen.value = false
-      mobileMenuOpen.value = false
-      store.dispatch('setUser', null)
-      router.push({ name: 'Home' })
+      try {
```

Comment:
This happens every once in a while to me where logout returns a 401 error due to a missing token in the db, then it is impossible to logout